src/components/Column/index.tsx

```diff
@@ -23,7 +23,7 @@ export const AutoColumn = styled.div<{
 export default Column
 
 export interface HideableAutoColumnProps {
-  show: boolean,
+  show: boolean
 }
```

Comment:
Let's find a solution for this convention in `.prettierrc`

Reply:
`'trailingComma':'es5'` => added to `.prettierrc` and fixed all files